nix/shell.nix

```diff
@@ -23,7 +23,7 @@ in mkShell {
       # build specific utilities
       clojure maven watchman
       # other nice to have stuff
-      yarn nodejs python310
+      yarn nodejs python310 babashka
```

Comment:
Better put `babashka` into the Clojure shell in `nix/shells.nix`
more info https://github.com/status-im/status-mobile/pull/17661#discussion_r1362256724

Reply:
Correct,

Reply:
done